web/app/test/ResourceListTest.jsx

```diff
@@ -0,0 +1,87 @@
+import _ from 'lodash';
+import Adapter from 'enzyme-adapter-react-16';
+import CallToAction from '../js/components/CallToAction.jsx';
+import ConduitSpinner from '../js/components/ConduitSpinner.jsx';
+import deployRollup from './fixtures/deployRollup.json';
+import Enzyme from 'enzyme';
+import ErrorBanner from '../js/components/ErrorBanner.jsx';
+import { expect } from 'chai';
+import MetricsTable from '../js/components/MetricsTable.jsx';
+import PageHeader from '../js/components/PageHeader.jsx';
+import { processSingleResourceRollup } from '../js/components/util/MetricUtils.js';
```

Comment:
Unused import?

Reply:
Good catch. Do we have this in eslint? I feel like some kind of tool should have yelled at me about it.

Reply:
Good question. Looks like our current eslint config catches this, but it's set to warn instead of error. So I think you would have seen the warning when running the tests, but the tests would still pass.

```
    "no-unused-vars": [1, { "args": "all", "argsIgnorePattern": "^_" }],
```

I'm working on an eslint branch, and I'll probably move all of our warn rules to error, to make it more obvious when code is in violation of one of the rules.